06_gpu_and_ml/stable_lm/main.py

```diff
@@ -0,0 +1,141 @@
+import time
+from pathlib import Path
+
+import torch
+from huggingface_hub import snapshot_download
+from pydantic import BaseModel, Field
+from rich import print
+from transformers import (
+    AutoModelForCausalLM,
+    AutoTokenizer,
+    GenerationConfig,
+    StoppingCriteria,
+    StoppingCriteriaList,
+    pipeline,
+)
+
+import modal
+
+
+def fetch_models():
+    MODEL_PATH = snapshot_download("stabilityai/stablelm-tuned-alpha-7b", ignore_patterns=["*.md"], revision="25071b093c15c0d1cb2b2876c6deb621b764fcf5")
+
+
+def compile_models():
+    from transformers import AutoModelForCausalLM, AutoTokenizer
+    MODEL_PATH = snapshot_download("stabilityai/stablelm-tuned-alpha-7b", ignore_patterns=["*.md"], revision="25071b093c15c0d1cb2b2876c6deb621b764fcf5")
+    m = AutoModelForCausalLM.from_pretrained(MODEL_PATH, torch_dtype=torch.float16, local_files_only=True).cuda()
+    tok = AutoTokenizer.from_pretrained(MODEL_PATH, local_files_only=True)
+    m = torch.compile(m)
+    m.save_pretrained(MODEL_PATH, safe_serialization=True, max_shard_size="24GB")
+    tok.save_pretrained(MODEL_PATH)
+    [p.unlink() for p in Path(MODEL_PATH).rglob("*.bin")]
```

Comment:
```suggestion
    [p.unlink() for p in Path(MODEL_PATH).rglob("*.bin")]  # type: ignore
```

Seeing `06_gpu_and_ml/stable_lm/run.py:34: error: "unlink" of "Path" does not return a value` from mypy

Reply:
Thanks for the heads up.